server/controllers/chartsController/testUtils.js

```diff
@@ -2,4 +2,783 @@ function postProcessDataFactory(mapEntries) {
   return new Map(Object.entries(mapEntries))
 }
 
-export { postProcessDataFactory }
+const labels = [
```

Comment:
should we move this diff over to the `test/fixtures.js` file?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/287/commits/53c75c19215f81ea1f646fb5e98102019a21521f